test/integration/manifests/cns/daemonset.yaml

```diff
@@ -131,4 +131,150 @@ spec:
             path: /run/xtables.lock
             type: File
       serviceAccountName: azure-cns
-
```

Comment:
So if we add windows daemonset to the same daemonset.yaml, when it tries to install CNS daemonset, installCNSDaemonset() just installs Linux CNS daemonset, can we set up the windows CNS conifgmap for azure cni overlay here?

Reply:
Ah, you're right, did not see this, thanks Paul